drift-transport-spi/src/main/java/io/airlift/drift/transport/client/MessageTooLargeException.java

```diff
@@ -17,10 +17,10 @@
 
 import io.airlift.drift.protocol.TTransportException;
 
-public class FrameTooLargeException
+public class MessageTooLargeException
```

Comment:
Just to avoid name clash. `FrameTooLargeException` now extends the `DecoderException`, and indicates max frame size overflow in FrameDecoder. `MessageTooLargeException` is a `TTransportException`.